lib/datasource/maven/index.ts

```diff
@@ -93,7 +93,8 @@ async function getDependencyInfo(
 }
 
 function getLatestStableVersion(versions: string[]): string | null {
-  const stableVersions = versions.filter(mavenVersion.isStable);
+  const { isStable } = mavenVersion; // auto this bind
```

Comment:
Was this not working before? Or does it work before and after?

Reply:
works both, but later does not cause lint error `@typescript-eslint/unbound-method`

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/unbound-method.md